ibm/resource_ibm_container_worker_pool_test.go

```diff
@@ -30,7 +30,7 @@ func TestAccIBMContainerWorkerPool_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"ibm_container_worker_pool.test_pool", "size_per_zone", "1"),
 					resource.TestCheckResourceAttr(
-						"ibm_container_worker_pool.test_pool", "labels.%", "3"),
+						"ibm_container_worker_pool.test_pool", "labels.%", "2"),
```

Comment:
Number of labels should decrease by because the system labels are now being ignored on read.